wordpress-mobile/WordPress-iOS
# Add Jetpack Secrets (Part 1)

This PR adds the foundation required to add Jetpack-specific secrets by replacing the old secrets file generation with a simple `.swift` file copy.

To test:
- Ensure that CI passes
- Build the WP Internal scheme locally (it should not fail)
- Build the WP Alpha scheme locally (it should not fail)
- Build the Jetpack scheme locally (it should not fail)
 
## Regression Notes
1. Potential unintended areas of impact
If the secrets were not copied into the new swift files correctly, it could break external services like Sentry, Google Login, Tenor, or WordPress.com.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I carefully checked that the old/new secrets were the same between the files

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
